### PR TITLE
Add codebyte exemples for python built in function delattr and eval

### PR DESCRIPTION
**Pull request by LaylaEBT**
### Description

-added codebytes for the terms eval and delattre in the python's built-in-function
-corrected the example line 47 of the code for eval
-closes issue #2571 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@LaylaEBT it looks like the email for "EL BAKALI TAHERI" needs to be added to your github profile.

**Review by SSwiniarski — APPROVED** [review]
@LaylaEBT I did a first review, made one minor change, and will pass this on to a second review.

File: content/python/concepts/built-in-functions/terms/eval/eval.md [hunk 1]
```diff
@@ -44,5 +44,20 @@ x = 10
...
+
+```codebyte/python
+x = 2
+y = 3
+
+print(eval("x * y")) 
+print(eval("x * y", {"x": 3, "y": y}))
+print(eval("min([-10, 20, 45])"))
+#the following example will create a TypeError as the first argument must be a string
+print(eval(x*y))
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/eval/eval.md:62]: Maybe we can include the type of the argument passed in within the syntax portion instead

Could you show this with a third argument?

**Review by yangc95 — COMMENTED** [review]
Hey @LaylaEBT , thanks for this addition! I left one comment to be addressed 🙂

**Review by yangc95 — APPROVED** [review]

**Pull request merged**